libs/shared/contentful/src/lib/contentful.service.ts

```diff
@@ -18,4 +26,16 @@ export class ContentfulService {
 
     return offeringResult.getOffering();
   }
+
+  async retrieveStripePlanId(
+    offeringConfigId: string,
+    interval: SubplatInterval
+  ) {
+    const planIds = await this.contentfulManager.getOfferingPlanIds(
+      offeringConfigId
+    );
+    const plan = await this.stripeManager.getPlanByInterval(planIds, interval);
```

Comment:
While testing `payments-next`, `getPlanByInterval` seems to reject with the following error.

`'You passed a non-recurring price to the Plan API. You must use the Price API instead.'`

Reply:
I've pushed up the following PR with a temporary solution. If you think this would work, feel free to merge it into this branch. https://github.com/mozilla/fxa/pull/16965